### ISSUE TITLE
Add ability to inspect Tailscale traffic

### ISSUE THREAD

**Issue by nickkeets** [ProxymanApp/Proxyman#2062]
## Description

It would be nice if Proxyman could integrate with [Tailscale](https://tailscale.com) so that traffic coming through tailscale can be inspected. 

## Why this feature/change is important?

The scenario I'm thinking is to simplify mobile app testing. Assuming all devices are running Tailscale, I could be running Proxyman on my Mac and whenever I want to inspect my phone's traffic I would switch to use the mac as an exit node. No need to change my WiFi settings.

**Comment by NghiaTranUIT** [comment]
@nickkeets I'm not sure how to do it, but if you would like to simply how to use Proxyman to capture your iPhone traffic, you can try [Atlantis](https://github.com/ProxymanApp/atlantis) (developed by Proxyman team).

- No proxy and no certificate needed.
- If you're in the same network, it just automatically discovers and works

**Comment by nickkeets** [comment]
Thanks, that's interesting. As an alternative to Tailscale I have also tried this solution: https://archive.is/24P1J#selection-1092.0-1149.17

You connect the device to your mac and create a network interface that you can then monitor with Wireshark. Of course that's on a lower level than Proxyman and can't decrypt HTTP traffic so not that useful.

If Proxyman could somehow inspect the traffic coming from that network interface would be ideal. But I don't know if this is possible, so feel free to close this issue if out of scope.

**Comment by NghiaTranUIT** [comment]
I don't think that it will work with Proxyman since Proxyman works under HTTP Layer.

Meanwhile, your solution is creating a virtual network, which is low-level that Proxyman can support. It's a reason why Wireshark works
